src/native/worker/RsaApp.js

```diff
@@ -1,34 +1,39 @@
 // @flow
 import {base64ToUint8Array, uint8ArrayToBase64} from "@tutao/tutanota-utils"
-import {nativeApp} from "../common/NativeWrapper"
-import {Request} from "../../api/common/WorkerProtocol"
+import {Request} from "../../api/common/Queue"
 import type {PrivateKey, PublicKey, RsaKeyPair} from "../../api/worker/crypto/RsaKeyPair"
+import type {NativeInterface} from "../common/NativeInterface"
+import type {RsaImplementation} from "../../api/worker/crypto/Rsa"
+import {random} from "../../api/worker/crypto/Randomizer"
```

Comment:
should we inject it?

Reply:
we added one level of injection, but Rsa.js already uses `random` so we will just leave that for now